File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/maps/StreetViewActivity.java
package com.jdroid.android.sample.ui.google.maps;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class StreetViewActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return StreetViewFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/google/maps/StreetViewActivity.kt
package com.jdroid.android.sample.ui.google.maps

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class StreetViewActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return StreetViewFragment::class.java
    }
}
